### DISCUSSION TITLE
对比v2.3.20，v2.4.x创建IVF_PQ索引性能相差~60倍

### DISCUSSION THREAD

**Discussion by xsmq** [milvus-io/milvus#34999] [Q&A and General discussion]
1、create_index参数
index_type: IVF_PQ
nlist: 128
m: 120
metric: L2

2、数据集
gist：100万条数据

3、cluster集群
node：3 * 16C64G
indexnode pod：3

4、性能对比
v2.3.20：创建索引时间 ~589s
v2.4.6：创建索引时间 ~35856s（v2.4.1和v2.4.5跑出来的结果同样很差）
性能劣化近60倍

**Comment by cydrain** [comment]
Milvus | Knowhere | Cardinal
---------|----------------|------------
v2.3.20 | v2.2.7 | 2.2.10
v2.4.6 | v2.3.5-hotfix | v2.4.5-hotfix

**Comment by cydrain** [comment]
File a issue https://github.com/zilliztech/knowhere/issues/739 to track this

**Comment by alexanderguzhva** [comment]
The correct solution is not to install `libopenblas`, but to install `libopenblas0-openmp` instead (or `libopenblas-openmp-dev`) for Ubuntu.

**Comment by liliu-z** [comment]
Solved by https://github.com/zilliztech/knowhere/pull/741
Will release this with Milvus 2.4.7

**Comment by xsmq** [comment]
it's great!!!
